shiny/templates/app-templates/04-dashboard-tips/_template.json

```diff
@@ -4,7 +4,7 @@
   "title": "Intermediate dashboard",
   "description": "An intermediate dashboard with value boxes, several plots in cards and a sidebar.",
   "next_steps": [
-    "Run the app with `shiny run app.py`."
+    "Run the app with `shiny run app.py` from the app directory."
```

Comment:
Actually, only the `basic-app` template needs this instruction, otherwise we already have instructions to move into the app directory. (https://github.com/posit-dev/py-shiny/pull/1631/commits/88d4c9f64f86b18427dfe629f31cfab286eded39)

```bash
❯ shiny create            
? Which template would you like to use? Intermediate dashboard
… Creating Intermediate dashboard Shiny app...
? Enter destination directory: ./04-dashboard-tips
? Would you like to use Shiny Express? Yes
✓ Created Shiny app at 04-dashboard-tips

→ Next steps:
- Install required dependencies:
    cd 04-dashboard-tips
    pip install -r requirements.txt
- Open and edit the app file: 04-dashboard-tips/app.py
- Run the app with `shiny run app.py`.

ℹ Just getting started with Shiny?
→ Learn more at https://shiny.posit.co/py/docs/user-interfaces.html
```

Reply:
FYI we keep the full path in

```
- Open and edit the app file: 04-dashboard-tips/app.py
```

because VS Code makes this path clickable. If we changed it to `app.py` we'd break click-to-open. But I think it's reasonable to expect the working directory change to carry over to the `shiny run` command.